evergreen.yml

```diff
@@ -105,37 +95,31 @@ functions:
           export COMPLIANCE_REPORT_NAME="mongo-odbc-driver_compliance_report.md"
           export STATIC_CODE_ANALYSIS_NAME="mongo-odbc-driver.sast.sarif"
           if [[ "${triggered_by_git_tag}" != "" ]]; then
-            # tag should be formatted as `v<ODBC major>.<minor>.<patch>-libv<libmongosqltranslate major>.<minor>.<patch>`
             # we split on `-libv` to get the two versions
-            export RELEASE_VERSION=$(echo ${triggered_by_git_tag} | awk -F'-libv' '{print $1}' | sed s/v// )
-            export LIBMONGOSQLTRANSLATE_VERSION=$(echo ${triggered_by_git_tag}  | awk -F'-libv' '{print $2}')
+            export RELEASE_VERSION=$(echo ${triggered_by_git_tag} | sed s/v// )
```

Comment:
[nit] Let's remove `# we split on -libv to get the two versions`. (Unfortunately, github won't let me suggest changes here)